abstracts/_mixins.scss

```diff
@@ -0,0 +1,32 @@
+@mixin respond($breakpoint) {
+  @if $breakpoint == phone {
+    @media only screen and (max-width: 37.5em) {
+      @content;
+    } //600px
+  }
+  @if $breakpoint == phone-sm {
```

Comment:
What does == mean or do?

Reply:
we use  @breakpoint as an argument for the mixin function and if the @breakpoint argument is equal to phone-sm means we gonna use that predefined styling.